dags/vz_cr3_extract_ocr_narrative.py

```diff
@@ -0,0 +1,72 @@
+import os
+import pendulum
+from datetime import datetime, timedelta
+
+from airflow.decorators import task
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Extracts the diagram and narrative out of CR3 pdfs",
+    "depends_on_past": False,
+    "start_date": pendulum.datetime(2019, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+docker_image = "atddocker/atd-vz-cr3-extract:production"
```

Comment:
noting that this came up in Charlie's metrobike PR https://github.com/cityofaustin/atd-airflow/pull/121#discussion_r1261630375 I know that this comes from existing patterns, and I think that I agree that it is nice to see the image name directly in the DockerOperator params? Also just realizing that I did this same thing in the service bot DAGs.